var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -73,6 +74,8 @@ class Vtk(CMakePackage):
 
     # VTK will need Qt5OpenGL, and qt needs '-opengl' for that
     depends_on('qt+opengl', when='+qt')
+    
+    depends_on('mpi', when='+mpi')
```

Comment:
do you know by coincidence which standard? MPI 2.3+? (`mpi@2.3:`)

Reply:
No sorry I have not found any info on this.